tests/core/htmldataprocessor/htmldataprocessor.js

```diff
@@ -30,6 +30,11 @@
 			output = output ? output.replace( /%xss%/g, 'testXss()' ) : input;
 
 		tcs[ name ] = function() {
+			// (#3483)
```

Comment:
I wonder if we shouldn't just try to fix this test (#3483) as it seems like the matter of porper HTML normalization before assertion?

Reply:
It's caused by changes in Safari engine - browsers were resigning from removing preceding spaces from attribute value (see #1070 and #1291 for reference), Safari was the only one left on that field, but it also adapted. I changed the test accordingly (actually removing browser sniffing instead of doing it twice, so it's a good change :joy: ).